Wrappers/Python/test/test_SumFunction.py

```diff
@@ -209,7 +206,72 @@ def test_SumFunction(self):
                 f1.gradient(tmp, out = out_right1)   
                 self.assertNumpyArrayAlmostEqual(out_left.as_array(), out_right1.as_array() + tmp_fun_gradient_out)
             except NotImplementedError:
-                print("{} is not differentiable".format(type_fun))  
+                # print("{} is not differentiable".format(type_fun))  
+                pass
+
+        ### test less than 2 functions
+        # list1
+
+        try:
+            f = SumFunction(list1[0])
+            self.assertFalse(True, "passed only one function and SumFunction accepted it!")
+        except ValueError as ve:
+            self.assertTrue(True, "Correctly failed" + str(ve))
+
+    def test_SumFunction2(self):
```

Comment:
Please split your test methods:
1) testing the `gradient` method
2) testing the `Lipschitz` constant

Reply:
Fixed